src/public/about.md

```diff
@@ -24,7 +26,7 @@ currently under development and will serve as the main European-level
 hub for data access, providing a user-friendly interface for
 researchers and healthcare professionals.
 
-## Key Objectives
+### Key Objectives
```

Comment:
It's generally better to follow a sequential heading structure in Markdown. # then ## instead of ###

Reply:
you are right, I was testing between the two font sizes and forgot to change them back, it is reverted now